src/modules/common/hooks/useCustomInfiniteQuery.js

```diff
@@ -29,13 +31,17 @@ export const useCustomInfiniteQuery = ({
   client = defaultClient
 }) => {
   const [{ chainID }] = useCurrentApplication();
+  const network = useSelector(state => state.network);
+  const networkName = getNetworkName(network)
+
   return useInfiniteQuery(
     [chainID, config, APPLICATION, METHOD, ...keys],
     async ({ pageParam }) => client[METHOD]({
       ...config,
       params: {
         ...(config.params || {}),
         ...pageParam,
+        network: networkName
```

Comment:
@soroushm @Balanced02 this is giving me concerns, because now, I am thinking, how are we going to handle the scenario if i want to switch to devnet. Which is a node I am running locally(meaning I can reach my node on localhost or 127.0.0.1). because we are passing the network name as a parameter. Are we expecting the service's server we are hitting to redirect the call based on the network params been passed. 

I think I need more clarification here